docs/examples/data-completeness.py

```diff
@@ -0,0 +1,91 @@
+"""
+Missing Data Periods
+====================
+
+Identifying days with missing data using a "completeness" score metric,
+and filtering out consecutive days with a low completeness score for
+future data analysis.
```

Comment:
This snippet gets used for the hovertext in the gallery index and therefore has to be kept relatively short. Hover over the thumbnails here and see how it gets cut off: https://pvanalytics--135.org.readthedocs.build/en/135/generated/gallery/index.html

Reply:
Updated in 
commit 6da6f0481f795b270164db6bc61988bb2fcb6fef